packages/components/src/Icons/custom/RemoveRounded.tsx

```diff
@@ -0,0 +1,6 @@
+import React from "react";
+import createSvgIcon from "../../SvgIcon/utils/createSvgIcon";
+
+export default createSvgIcon(
+  <path d="M3.75 12.25C3.75 12.8 4.2 13.25 4.75 13.25H18.75C19.3 13.25 19.75 12.8 19.75 12.25C19.75 11.7 19.3 11.25 18.75 11.25H4.75C4.2 11.25 3.75 11.7 3.75 12.25Z" />,
```

Comment:
I made this one, so this path doesn't exist in `traject` or in the UI kit. Here's the blocky one in traject: https://github.com/FB-PLP/traject/blob/77d303699345622a3f8e0aebe013fbf3c1a7d421/app/assets/javascripts/v2/core/EDSCandidates/SvgIcon/Remove.jsx